build.sbt

```
@@ -90,3 +90,16 @@ lazy val circe = crossProject(JSPlatform, JVMPlatform, NativePlatform)
     )
   )
   .dependsOn(core, testkit % Test)
+
+lazy val benchmarks = crossProject(JVMPlatform)
+  .crossType(CrossType.Pure)
+  .in(file("benchmarks"))
+  .enablePlugins(NoPublishPlugin, JmhPlugin)
+  .settings(commonSettings: _*)
+  .settings(
+    name := "diffson-benchmarks",
+    libraryDependencies ++= Seq(
+      "io.circe" %% "circe-literal" % circeVersion
```

```suggestion
      "io.circe" %% "circe-core" % circeVersion
```
You don't seem to use `literal`, just syntax from `core`.